KonomiAI/Chronicle
# feat: form viewer

### Related Issue(s)



resolves #177 



### Quick overview of the change
Form Viewer which takes in form object from mongodb and renders the visual representation for the form

Included some basic db seeding for forms for testing purposes

Built test page for form viewer

### Quick testing guide

1. seed the mongoDB
2. `docker-compose up -d` on root dir
3. `docker-compose --build db_seed`
4. after seeing a successful seed, go to [`localhost:5555`](http://localhost:5555)
5. in primsa studio, check for seeded Form and copy the `id`
6. remaining in the Form, click on the `lastestFormVersion` and attach the only seeded FormVersion by clicking the checkbox
7. save changes in prisma studio
8. Now in terminal, `cd client`
9. `yarn dev`
10. in `src/pages/forms/ViewForm.tsx`, paste in the Form `id` copied from before
11. go to [`localhost:3000/forms/testViewForm`](http://localhost:3000/forms/testviewform)
